src/librustdoc/html/static/css/settings.css

```diff
@@ -6,17 +6,10 @@
 .setting-line > div {
 	display: inline-block;
 	vertical-align: top;
-	font-size: 17px;
+	font-size: 1.06rem;
```

Comment:
This is a nice change to use rem. But is there a good reason to have this font size slightly larger than the default? I think it would be fine at 1rem (i.e. don't override the default here).

Reply:
Done.